### PR TITLE
(ci-skip) Fix comment typos.

### PR DESCRIPTION
**Pull request by AdamHillier**

### CODE REVIEW

**Comment by AdamHillier** [comment]
@lgeiger I thought the `(ci-skip)` in the commit message would stop CI running?

**Comment by lgeiger** [comment]
> I thought the (ci-skip) in the commit message would stop CI running?

Looks like this doesn't work anymore since we moved to run CI on PRs instead of on push :(

File: larq_compute_engine/core/bgemm_kernels_arm64.h [hunk 1]
```diff
@@ -552,12 +552,12 @@ void CheckOffsetsInKernelParams64BP(const Params&) {
...
 // v4 -- v7 from RHS:
 //
-//                                      int32 RHS 2x4 block
+//                                    int64 RHS 2x4 block
 //                          /--------------------------------------\
 //                          |v4.d[0]         ...          v7.d[0]  |
 //                          |v4.d[1]         ...          v7.d[1]  |
 //                          \--------------------------------------/
-//    int32 LHS 4x2 block
+//     int64 LHS 4x2 block
```
Comment (AdamHillier) [inline larq_compute_engine/core/bgemm_kernels_arm64.h]: @arashb/@tombana I could be wrong about this but I think these are int64 blocks (here and line 555)?

**Comment by AdamHillier** [comment]
Also it doesn't matter much but [line 135](https://github.com/larq/compute-engine/blob/f561da84157433b2cf23611a38e46827493190d9/larq_compute_engine/core/bgemm_kernels_arm64.h#L135) conflicts with the [diagram of the accumulator block](https://github.com/larq/compute-engine/blob/f561da84157433b2cf23611a38e46827493190d9/larq_compute_engine/core/bgemm_kernels_arm64.h#L147-L150).

File: larq_compute_engine/core/bgemm_kernels_arm64.h (hunk 1 above)
Comment (arashb) [reply larq_compute_engine/core/bgemm_kernels_arm64.h]: you are right, it should be int64.

**Comment by arashb** [comment]
> Also it doesn't matter much but [line 135](https://github.com/larq/compute-engine/blob/f561da84157433b2cf23611a38e46827493190d9/larq_compute_engine/core/bgemm_kernels_arm64.h#L135) conflicts with the [diagram of the accumulator block](https://github.com/larq/compute-engine/blob/f561da84157433b2cf23611a38e46827493190d9/larq_compute_engine/core/bgemm_kernels_arm64.h#L147-L150).
> 
> The latter doesn't look right to me regardless because it shows `v16 ... v22` which is 7 registers but I think it should be either 4 or 8?

The comment is wrong but the diagram is correct. the destination registers are the following 4 registers: v16, v18, v20, v22.

**Pull request merged**